script.js

```diff
@@ -1,4 +1,120 @@
+    return `Hello ${userName}, are you ready to play?`;
+  } else if (gameMode == "GAME ON") {
+    randomHandType = generateRandomHand();
+    message = getComputerHand(randomHandType, input);
+    return message;
+  }
+};
+
+//this is where you're defining computerHand as a function. missing 'function' in line 50
+var getComputerHand = function (computerHand, userGuess) {
+  console.log("Computer chooses:" + computerHand);
+  totalGames = totalGames + 1;
+  var genericOutput = `Your hand: ${userGuess}. Computer's Hand: ${computerHand}.`;
+  if (userGuess == computerHand) {
+    drawCount += 1;
+    winningPercentage = (winCount / totalGames) * 100;
+    var message = `${genericOutput} DRAW!! <br> You have won ${winCount} times. <br> You have lost ${lossCount} times. <br> You have ${drawCount} draws. <br> Your winning percentage is ${winningPercentage}%`;
+  }
+  if (userGuess == "scissors" && computerHand == "paper") {
+    winCount += 1;
+    winningPercentage = (winCount / totalGames) * 100;
+    var message = `${genericOutput} WIN!!! <br> You have won ${winCount} times. <br> You have lost ${lossCount} times. <br> You have ${drawCount} draws. <br> Your winning percentage is ${winningPercentage}%`;
+  }
+  if (userGuess == "scissors" && computerHand == "stone") {
+    lossCount += 1;
+    winningPercentage = (winCount / totalGames) * 100;
+    var message = `${genericOutput} LOSE!! <br> You have won ${winCount} times. <br> You have lost ${lossCount} times. <br> You have ${drawCount} draws. <br> Your winning percentage is ${winningPercentage}%`;
+  }
+  if (userGuess == "paper" && computerHand == "stone") {
+    winCount += 1;
+    winningPercentage = (winCount / totalGames) * 100;
+    var message = `${genericOutput} WIN!!! <br> You have won ${winCount} times. <br> You have lost ${lossCount} times. <br> You have ${drawCount} draws. <br> Your winning percentage is ${winningPercentage}%`;
+  }
+  if (userGuess == "paper" && computerHand == "scissors") {
+    lossCount += 1;
+    winningPercentage = (winCount / totalGames) * 100;
+    var message = `${genericOutput} LOSE!! <br> You have won ${winCount} times. <br> You have lost ${lossCount} times. <br> You have ${drawCount} draws. <br> Your winning percentage is ${winningPercentage}%`;
+  }
+  if (userGuess == "stone" && computerHand == "scissors") {
+    winCount += 1;
+    winningPercentage = (winCount / totalGames) * 100;
+    var message = `${genericOutput} WIN!!! <br> You have won ${winCount} times. <br> You have lost ${lossCount} times. <br> You have ${drawCount} draws. <br> Your winning percentage is ${winningPercentage}%`;
+  }
+  if (userGuess == "stone" && computerHand == "paper") {
+    lossCount += 1;
+    winningPercentage = (winCount / totalGames) * 100;
+    var message = `${genericOutput} LOSE!! <br> You have won ${winCount} times. <br> You have lost ${lossCount} times. <br> You have ${drawCount} draws. <br> Your winning percentage is ${winningPercentage}%`;
+  }
```

Comment:
Good use of if statements. However do remember that you can use the or operator to simply the conditions here. 
For example the condition to win can be written like this : 

```
  if (userGuess == "scissors" && computerHand== "paper" || userGuess == "paper" && computerHand== "stone" || userGuess == "stone" && computerHand== "scissors" ){
      //win
  }
```